src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -82,6 +83,11 @@ export default App.extend(extend({
 
     Radio.request('ws', 'add', model);
   },
+  onAttachmentAdded(model) {
+    this.attachments.add(model);
+
+    Radio.request('ws', 'add', model);
+  },
```

Comment:
This `onAttachmentAdded()` function name is almost identical to the already existing `onAddAttachment()` in this file.

We should probably name this something else, like `OnWsAttachmentAdded()`.